contrib/guix/patches/glibc-2.24-elfm-loadaddr-dynamic-rewrite.patch

```diff
@@ -0,0 +1,60 @@
+commit 6b02af31e9a721bb15a11380cd22d53b621711f8
```

Comment:
Is there a source for this commit?


https://sourceware.org/git/?p=glibc.git;a=commit;h=6b02af31e9a721bb15a11380cd22d53b621711f8 is a 404

Reply:
https://sourceware.org/git/?p=glibc.git;a=commit;h=a68ba2f3cd3cbe32c1f31e13c20ed13487727b32